src/test/obj_ctl_arenas/obj_ctl_arenas.c

```diff
@@ -242,6 +258,64 @@ main(int argc, char *argv[])
 		ret = pmemobj_ctl_get(pop, "heap.narenas.total", &narenas_a);
 		UT_ASSERTeq(ret, 0);
 		UT_ASSERTeq(narenas_b + narenas_n, narenas_a);
+	} else if (t == 'f') {
+		int ret;
```

Comment:
Move this test into a separate function and run that function in multiple threads. And then add a helgrind force-enabled test.